sdk/storage/storage-blob/src/sas/AccountSASSignatureValues.ts

```diff
@@ -90,6 +90,13 @@ export function generateAccountSASQueryParameters(
   accountSASSignatureValues: AccountSASSignatureValues,
   sharedKeyCredential: StorageSharedKeyCredential,
 ): SASQueryParameters {
+  return generateAccountSASQueryParametersInternal(accountSASSignatureValues, sharedKeyCredential).sasQueryParameters;
+}
+
+export function generateAccountSASQueryParametersInternal(
```

Comment:
Optional : To make sure generate stringTosign function is well tested, It's better to make generate SAS function call generate stringTosign function, then SAS test will cover stringTosign function.

Or could we find other ways to make generate stringTosign function has test coverage? like sign it and compare with the generated SAS?

Reply:
I will added more test cases later.